jmclient/jmclient/wallet_utils.py

```diff
@@ -1034,29 +1126,99 @@ def wallet_freezeutxo(wallet, md, display_callback=None, info_callback=None):
                           .format(fmt_utxo((txid, index))))
     return "Done"
 
-def get_wallet_type():
+
+
+def wallet_gettimelockaddress(wallet, locktime_string):
```

Comment:
Why isn't this a method of `FidelityBondMixin` ? It seems like the only thing that would change is the need to do the `isinstance` check.
I believe the same question applies to the following method `wallet_addtxoutproof`

Reply:
I followed the convention set by `wallet_signmessage` `wallet_dumpprivkey` `wallet_showseed`.

Reply:
Understood. We should look into this perhaps, but not an issue that needs to be addressed here.